config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  get 'bookings/create'
```

Comment:
Don't think we should have this routs once we are using resources!

Reply:
just saw them will delete and redo the pull request